js/scenes/ProjectResourceTest/ProjectResourceTestContainer.js

```diff
@@ -19,12 +19,18 @@ class ProjectResourceTestContainer extends Component {
     }
   };
   componentDidMount() {
-    this.props.dispatch(getCompanyQuestions("adobe", "indesign"));
+    this.props.dispatch(
+      getCompanyQuestions(
+        this.props.route.params.props.company,
+        this.props.route.params.props.project
```

Comment:
you can deconstruct company and project here instead of passing ```this.props.route.params.props.project```

Reply:
This creates a yellow box that says the new props are undefined. I can console.log them and they are defined. The component still works. I don't understand how they could be undefined.

Reply:
Although they kinda should be required, I've alleviated the problem by not requiring them.